projects/aca-testing-shared/src/components/header/header.ts

```diff
@@ -33,7 +33,7 @@ import { BrowserActions } from '@alfresco/adf-testing';
 
 export class Header extends Component {
   logoLink = this.byCss('.app-menu__title');
-  moreActions = browser.element(by.id('app.header.more'));
+  moreActions = this.byCss(`.userMenuButton`);
```

Comment:
variable name does not match the selector, should be called `userMenuButton` instead of `moreActions`

Reply:
Got it, I've renamed it